src/button.scss

```diff
@@ -0,0 +1,81 @@
+@import "./variables/button.scss";
+
+/*
+ * Button classes
+ * Component code : bt
+ *
+ * Modifier 1     : s   ( small size  button )
+ *                  m   ( normal size button )
+ *                  l   ( large size button )
+ *                  py  ( primary button )
+ *                  sy  ( secondary button )
+ *                  ol  ( outline button )
+ *                  tx  ( text button )
+ *                  gy  ( grey button )
+ *                  dis ( disable button )
+ */
+
+.bt-s { padding: 2.5px 7.5px; }
+.bt-m { padding: 5px 15px; }
+.bt-l { padding: 10px 30px; }
```

Comment:
No hardcoding of values anywhere please! You can use the SCSS variables to carry these numbers.